src/Pagination.js

```diff
@@ -111,12 +117,16 @@ Pagination.propTypes = {
   children: PropTypes.node,
   className: PropTypes.string,
   maxButtons: PropTypes.number,
-  onSelect: PropTypes.func
+  onSelect: PropTypes.func,
+  leftBtn: PropTypes.node.isRequired,
+  rightBtn: PropTypes.node.isRequired
```

Comment:
are these two required?

Reply:
Yes, they are; When the developer doesn't provide them as `prop` they must be there no matter what.
For this reason I've also added 

```javascript
{
  ...
  leftBtn: <Icon>chevron_left</Icon>,
  rightBtn: <Icon>chevron_right</Icon>,
}
```

to `Pagination.defaultProps`

Reply:
if `defaultProps` is present that means they are not required AFAIK

Reply:
Well, if you think about it it make sense to use `isRequired` coupled with `defaultProps`.

This component require these two `prop`s no matter what, and if you do not provide them the `default` one are going to be used (which, by the way, are the icons used by `materialize-css`).

As you can see I'm using `defaultProps` as a way to enforce that these two `prop`s are there.

Hope I explained my self well :)

Reply:
I still don't quite understand. AFAIK `isRequired` means that without the prop, the component will break. It is not required because even if I don't pass the prop, it will use a default prop.

Reply:
Alright then, going to remove `isRequired` :)